news/utils.py

```diff
@@ -321,27 +314,22 @@ def get_best_language(languages):
     return languages[0]
 
 
-def validate_email(email):
+def process_email(email):
     """Validates that the email is valid.
 
-    Returns None on success and raises a ValidationError if
-    invalid. The exception will have a 'suggestion' parameter
-    that will contain the suggested email address or None.
-
-    @param email: unicode string, email address hopefully
-    @return: None if email address in the 'email' key is valid
-    @raise: EmailValidationError if 'email' key is invalid
+    Return email ascii encoded if valid, None if not.
     """
-    # disabling this for now and falling back to dumb regex validation.
-    # Bug 1066762.
-    # TODO: Find a more robust solution
-    #       Possibly ET's email validation API.
+    if not email:
+        return None
+
+    email = force_unicode(email)
     try:
-        dj_validate_email(force_unicode(email))
-    except ValidationError:
-        raise EmailValidationError('Invalid email address')
+        info = validate_email(email, allow_smtputf8=False,
```

Comment:
nit: add a comment to remind us that Salesforge doesn't support SMPTUTF8 and a test to prevent accidental activation in the future?